docs/OV_Runtime_UG/stateful_models_intro.md

```diff
@@ -0,0 +1,90 @@
+ between inferences. Kaldi contains special specifier `Offset` to define time offset in a network. 
+ 
+ OpenVINO also contains special API to simplify work with networks with states. State is automatically saved between inferences, 
+ and there is a way to reset state when needed. You can also read state or set it to some new value between inferences.
+ 
+## OpenVINO State Representation
+
+ OpenVINO contains a special abstraction `Variable` to represent a state in a network. There are two operations to work with the state: 
+* `Assign` to save value in state
+* `ReadValue` to read value saved on previous iteration
+
+![state_network_example](./img/state_network_example.png)
+
+You can find more details on these operations in [ReadValue specification](../ops/infrastructure/ReadValue_3.md) and 
+[Assign specification](../ops/infrastructure/Assign_3.md).
+
+## How to get the OpenVINO Model with states
+
+* [Convert Kaldi model to IR via Model Optimizer.](../MO_DG/prepare_model/convert_model/kaldi_specific)
+   If the original Kaldi model contains RNN-like operations with `stateful` option, then after ModelOptimizer conversion,
+   the resulting OpenVINO model will also contain states, represented by pairs ReadValue/Assign operations.
+
+* [Apply LowLatency2 transformation.](./ways_to_get_stateful_model.md#)
+   If after conversion from original model to OpenVINO representation, the resulting model contains TensorIterator or Loop operations,
+   the LowLatency2 transformation can be applied to get model with states.
+   Note: there are some [specific limitations]() to use the transformation.
+
+* [Apply MakeStateful transformation.](./ways_to_get_stateful_model.md)
+   If after conversion from original model to OpenVINO representation, the resulting model contains Parameter and Result operations,
+   which pairwise have the same shape and element type, the MakeStateful transformation can be applied to get model with states.
+
+* [Create the model via OpenVINO API.](./ways_to_get_stateful_model.md)
+   For testing purposes or for some specific cases, when the ways to get OpenVINO model with states described above are not enough for your purposes,
+   you can use OpenVINO API and create `ov::opset8::ReadValue` and `ov::opset8::Assign` operations directly.
+
+## OpenVINO State API
+
+ Inference Engine has the `ov::InferRequest::query_state` method  to get the list of states from a network and `ov::VariableState` class to operate with states. 
+ Below you can find brief description of methods and the workable example of how to use this interface.
+ 
+ * `std::string get_name() const`
+   returns name(variable_id) of according Variable
+ * `void reset()`
+   reset state to default value
+ * `void set_state(const Tensor& state)`
+   set new value for state
+ * `Tensor get_state() const`
+   returns current value of state
```

Comment:
This is a kind of reference information. Can we build story based on examples, where we describe the purpose of each method instead of this list. At the moment when this list is provided it is unclear which class has this methods and why I need to call them, so this information will skip mind of the user until the real example appears.